examples/pipelines/simple_pipeline/components/image_filtering/src/main.py

```diff
@@ -18,19 +18,19 @@ class ImageFilterComponent(FondantTransformComponent):
     Component that filters images based on height and width.
     """
 
-    def transform(self, dataframe: dd.DataFrame, args: Dict) -> dd.DataFrame:
+    def transform(self, dataframe: dd.DataFrame, *, min_width: int, min_height: int) -> dd.DataFrame:
         """
         Args:
             df: Dask dataframe
```

Comment:
```suggestion
            df: The Dask dataframe to load as defined in the component specification 
```
Also make sure to include this doc in the `transform` method of the component class (see below)